test/integration/slickgrid_test.rb

```diff
@@ -0,0 +1,189 @@
+require 'test_helper'
+
+class SlickgridTest < ActionDispatch::IntegrationTest
+  include CapyHelper
+
+  self.use_transactional_fixtures = false
+
+  compare_data = [
+    { '100' => 'D', '101' => 'A', '102' => '4', '103' => '01/01/1991 01:01:01', '104' => '3', '105' => '3' },
+    { '100' => 'E', '101' => 'B', '102' => '5', '103' => '01/02/1992 02:02:02', '104' => '4', '105' => '4' },
+    { '100' => 'F', '101' => 'C', '102' => '6', '103' => '01/03/1993 03:03:03', '104' => '5', '105' => '5' },
+    { '100' => 'G', '101' => 'A', '102' => '7', '103' => '01/04/1994 04:04:04', '104' => '6', '105' => '6' }
+  ]
+
+  setup do
+    Capybara.current_driver = :webkit
+  end
+
+  teardown do
+    finish
+  end
+
+  def assert_similar_arrays(a, b)
+    c = a - b
+    d = b - a
+
+    assert c.length + d.length == 0, "Arrays\n\n#{a}\n\nand\n\n#{b}\n\ndo not have the same contents: #{c + d}"
```

Comment:
`assert_similar_arrays` now lives in two files.  This could definitely moved to a common file that both test scripts include.